.github/ISSUE_TEMPLATE/task.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Task
+about: Document a task to be done by the development team. It does not change the software.
+title: ''
+labels: task
+assignees: ''
+
+---
+
+<!--
+If you've never submitted an issue to the SORMAS repository before or this is your first time using this template, please read the Contributing guidelines (https://github.com/hzi-braunschweig/SORMAS-Project/blob/development/docs/CONTRIBUTING.md) for an explanation of the information we need you to provide. You don't have to remove this comment or any other comment from this issue as they will automatically be hidden.
+-->
+### Problem Description
```

Comment:
=> maybe rename to Requirement Description (I'd also be fine with Problem Description though)

Reply:
My association is: _Requirement_ relates to _Requirements Engineering_ relates to _feature_ / _story_ etc. I will keep "problem" to be solved.